File: second/src/test/java/com/pauldaniv/template/lib/second/consume/UserTest.java
package com.pauldaniv.template.lib.second.consume;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;

public class UserTest {

    @Test
    public void createDeployment() {
        String first = "testOne, ";
        String second = "testTwo";
        User user = new User(first, second);
        assertThat(user.doTest()).isEqualTo(first + second);
    }
}

File: second/src/test/kotlin/com/pauldaniv/template/lib/second/consume/UserTest.kt
package com.pauldaniv.template.lib.second.consume

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class UserTest {

  @Test
  fun createDeployment() {
    val first = "testOne, "
    val second = "testTwo"
    val user = User(first, second)
    assertThat(user.doTest()).isEqualTo(first + second)
  }
}
